### PR TITLE
Fix pam and config file handling

### PR DESCRIPTION
**Pull request by M4rtinK**
Looks like this was not done correctly during the recent build system migration, lets fix that.

Also fix Glade dep on ELN.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -43,10 +43,17 @@ default: all
 
 all: po-files
 
+install-non-python-stuff:
+	install -d $(DESTDIR)/etc/initial-setup/conf.d/
+	install -m 644 data/10-initial-setup.conf $(DESTDIR)/etc/initial-setup/conf.d/10-initial-setup.conf
+	install -d $(DESTDIR)/etc/pam.d/
+	install -m 644 pam/initial-setup $(DESTDIR)/etc/pam.d/initial-setup
+
```
Comment (vojtechtrefny) [inline Makefile]: This is already being installed in SPEC: https://github.com/rhinstaller/initial-setup/blob/bef4bb1a77263c6c4487a6ed0e5f79b7c8d64490/initial-setup.spec#L150

I'd prefer having this here in the Makefile, but there is no reason to have it twice.

**Review by vojtechtrefny — APPROVED** [review]
Looks good to me.

The ELN build fails because `python3-wheel` is not available. I think `wheel`is no longer needed since #159 so it might be possible do remove the dependency from `pyproject.toml` now.

**Comment by M4rtinK** [comment]
Dropped `python-wheel`, lets see what happens.

**Comment by vojtechtrefny** [comment]
One more partially related thing: Copr builds on PRs are enabled only for fedora-eln: https://github.com/rhinstaller/initial-setup/blob/bef4bb1a77263c6c4487a6ed0e5f79b7c8d64490/.packit.yml#L19-L23

It would be nice to enable these for rawhide and other versions as well so we can see the changes actually work. This is also why the tests never finish -- they are waiting for the rawhide RPMs, but there are none :-)

**Comment by M4rtinK** [comment]
> One more partially related thing: Copr builds on PRs are enabled only for fedora-eln:
> 
> https://github.com/rhinstaller/initial-setup/blob/bef4bb1a77263c6c4487a6ed0e5f79b7c8d64490/.packit.yml#L19-L23
> 
> It would be nice to enable these for rawhide and other versions as well so we can see the changes actually work. This is also why the tests never finish -- they are waiting for the rawhide RPMs, but there are none :-)

Ooops - lets another commit with this. :D
